### PR TITLE
Create enumerable

### PR DESCRIPTION
**Pull request by Dadadon**
- Created a class MyList that has an instance variable @list
- In MyList implemented a method #each that yields successive members of @list and uses the MyEnumerable module
**Created a module MyEnumerable that implements the following methods**
- #all?
- #any?
- #filter
- Each class and module has a separate .rb file

### CODE REVIEW

**Comment by omarsalem7** [comment]
Hi @Dadadon 😊, 

While you made a great effort in this project, I cannot proceed to review your code.

## Invalid Code Review Request 🛑 💀 

You have submitted a project, that doesn't have an updated README template

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by Ismailco — COMMENTED** [review]
Hi @Dadadon, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has an old README template, kindly make sure to use the [new README template provided by microverse](https://github.com/microverseinc/readme-template)

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: my_list.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+require './my_enumerable'
+
+class MyList
+  include MyEnumerable
+
+  def initialize
+    @list = []
+  end
```
Comment (youmari) [inline my_list.rb]: - [ ] when I tried to verify your app with an example in the requirements it doesn't work cause you didn't add a way to accept multiple arguments so please make sure to add that in order to verify you app is working 
```ruby 
# Create our list
list = MyList.new(1, 2, 3, 4)

# Test #all?
p list.all? {|e| e < 5}
p list.all? {|e| e > 5}

# Test #any?
p list.any? {|e| e == 2}

p list.any? {|e| e == 5}

# Test #filter
p list.filter {|e| e.even?}

=> #<MyList: @list=[1, 2, 3, 4]>
```

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @Dadadon , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ Great commit messages
✔️ PRO PR
✔️ No linter error 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by cynthiainga — APPROVED** [review]
## STATUS APPROVED ✔️ ✔️ 

Hi @Dadadon, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- [ ] No linters errors ✔️ 
- [ ] Descriptive PR ✔️ 
- [ ] Good commit history ✔️ 
- [ ] Correct Gitflow ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**